config/irsl/irsl_assembler_config.yaml

```
@@ -3623,6 +3623,1318 @@ PartsSettings:
       types : [PC1]
       translation: [48.0, 8.0, 0.0]
       rotation: [1., 0., 0., 180.0]
+  -
+    type: vehicle-base-ps
+    class: Base plate (Plag and socket)
+    description: Base plate for vehicles with LEGO plugs on both sides
```

両面プラグであるという説明のままになっているため、以下の修正案はいかがでしょうか。

```suggestion
    description: Base plate for vehicles with LEGO plugs and sockets on each side.
```

---

記載修正します．